marlowe-runtime/marlowe-tx/Main.hs

```diff
@@ -18,16 +18,14 @@ import Data.Version (showVersion)
 import Language.Marlowe.Runtime.ChainSync.Api (BlockNo(..), ChainSyncQuery(..), RuntimeChainSeekClient)
```

Comment:
Changes in this file:

- Move server sources to component outputs
- Connection refactor